client/extensions/woocommerce/state/selectors/test/promotions.js

```diff
@@ -19,8 +19,10 @@ describe( 'promotions', () => {
 			woocommerce: {
 				ui: {
 					promotions: {
-						currentPage: 12,
-						perPage: 42,
+						list: {
+							currentPage: 12,
+							perPage: 42,
```

Comment:
Suggestion - use a more real-world value for perPage like 10 (or 30?)

Reply:
It's 30 now from a previous PR